modules/ding_webtrekk/js/ding_webtrekk.opt-out.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * Implement opt-out option for the Webtrekk cookie.
+ */
+
+(function($) {
+
+  "use strict";
+
+  Drupal.behaviors.ding_webtrekk = {
+    attach: function(context) {
+      // If Webtrekk is not setup correctly, we might not have this.
+      if (typeof webtrekkV3 === 'undefined') {
+        return;
+      }
```

Comment:
I suggest moving this to the `click` handler itself. This will give the browser additional time to load webtrekk.

Reply:
Done :+1: